modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -38,14 +38,46 @@ $ oc get Network.config.openshift.io cluster -o yaml > cluster-openshift-sdn.yam
 [source,terminal]
 ----
 $ oc patch Network.operator.openshift.io cluster --type='merge' \
-  --patch '{ "spec": { "migration": {"networkType": "OVNKubernetes" } } }'
+  --patch '{ "spec": { "migration": { "networkType": "OVNKubernetes" } } }'
 ----
 +
 [NOTE]
 ====
 This step does not deploy OVN-Kubernetes immediately. Instead, specifying the `migration` field triggers the Machine Config Operator (MCO) to apply new machine configs to all the nodes in the cluster in preparation for the OVN-Kubernetes deployment.
 ====
 
+. Optional: You can enable automatic migration of several OpenShift SDN capabilities to the OVN-Kubernetes equivalents:
++
+--
+* Egress IP addresses
+* Egress network policies
+* Multicast
+--
++
+To enable automatic migration of the configuration for any of the previously noted OpenShift SDN features, specify the following keys:
++
++
+[source,terminal]
```

Comment:
The API is here https://github.com/openshift/api/blob/0b39f81154fa42abe26933181141ed4b4e3ad247/operator/v1/0000_70_cluster-network-operator_01.crd.yaml#L376

So it should be:

```
      "migration": {
        "networkType": "OVNKubernetes",
        "features": {
          "egressIP": <bool>,
          "egressFirewall": <bool>,
          "multicast": <bool>
        }
      }
```